src/object/interrupt.c

```diff
@@ -182,21 +182,22 @@ void deletedIRQHandler(irq_t irq)
 void handleInterrupt(irq_t irq)
 {
     if (unlikely(IRQT_TO_IRQ(irq) > maxIRQ)) {
-        /* mask, ack and pretend it didn't happen. We assume that because
-         * the interrupt controller for the platform returned this IRQ that
-         * it is safe to use in mask and ack operations, even though it is
-         * above the claimed maxIRQ. i.e. we're assuming maxIRQ is wrong */
-        printf("Received IRQ %d, which is above the platforms maxIRQ of %d\n", (int)IRQT_TO_IRQ(irq), (int)maxIRQ);
+        /* The interrupt number is out of range. Pretend it did not happen by
+         * handling it like an inactive interrupt (mask and ack). We assume this
+         * is acceptable, because the platform specific interrupt controller
+         * driver reported this interrupt. Maybe the value maxIRQ is just wrong
+         * or set to a lower value because the interrupts are unused.
+         */
+        printf("Received IRQ %d, which is above the platforms maxIRQ of %d\n",
+               (int)IRQT_TO_IRQ(irq), (int)maxIRQ);
```

Comment:
It's not really a big problem here (Ok to leave it from my side), but in general there is not reason to wrap this line. The code wrap column for C is [120] and the previous one was correct. If it wasn't the style checker would have complained. In general rewrapping code should be avoided to reduce conflicts with other PRs.

[120]: https://github.com/seL4/seL4_tools/blob/6211394b3f83eed47a2abe80b81af608846cdb66/misc/astylerc#L20

Reply:
reverted